packages/shared/ReactFeatureFlags.js

```diff
@@ -215,6 +215,7 @@ export const disableClientCache = true;
 // during element creation.
 export const enableRefAsProp = true;
 export const disableStringRefs = true;
+export const logStringRefs: null | ((string, string) => void) = null;
```

Comment:
Can you rename this to have the `enable` prefix, and call out "prod"? like `enableLogStringRefsProd`